letsdrill.rb

```diff
@@ -1,13 +1,39 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+=begin
+	grade="A"
+	if integer<60
+		grade ="F"
+	elsif integer<70
+		grade ="D"
+	elsif integer < 80
+		grade = "C"
+	elsif integer <90
+		grade="B"
+	end	
+	return grade
+=end
+
+	return case integer
+	when 0..59
+		"F"
+	when 60..69
+		"D"
+	when 70..79
+		"C"
+	when 80..89
+		"B"
+	else
+		"A"
+	end
 
 end
 
 def shortest_string(array)
-
-  #Put your code here!
-
+	short = array.first
+	array.each do |word|
+		short=word if word.length<short.length
+	end
+	return short 
```

Comment:
This works.  But take a look through the enumerable methods and see if you can't solve it with a one-liner.